### ISSUE TITLE
Internal anchor to bibliographic items lost

### ISSUE THREAD

**Issue by stijnvanhoey** [inbo/tutorials#73]
When using `.md` as the input for the website itself and knitting `Rmd` to `md`, the option of `link-citations: yes` is not working anymore for bibliographic items, as the knitting to markdown is not providing identifiers to the individual bibliographic items.

Options:
- exclude the `link-citations: yes` option and assume people will check the bibliography (easy fix)
- local fix on file itself that can be run after knitting to adapt the resulting markdown according to hugo-syntax and as such supporting the internal linking
- provide special routine within the website building process that overcomes this issue?!?

**Comment by florisvdh** [comment]
I'd recommend your quick fix. As long as the citations and bibliography render, all is OK. Citation links only become useful with long bibliographies, which will probably not be the case in the website.

**Comment by stijnvanhoey** [comment]
I agree, thanks. See https://github.com/inbo/tutorials/pull/78/commits/0dea28cf5926579f206c211b848b7705e3a90b5e for the adaptations
